test/integration/relations.test.js

```diff
@@ -85,9 +86,9 @@ describe('Relations', function () {
     });
 
     describe('Many-to-Many', function () {
-        before(utils.db.init('relations1'));
+        before(utils.db.init('many-to-many'));
 
-        describe('EQUALS', function () {
+        describe('EQUALS $eq', function () {
```

Comment:
Couldn't decide which one to go with, just the $eq or both, so stayed with both for now. Introduced this naming to be able to search easier groups of test suites  (`equal` or just `$eq` are used all over the place and it was hard to find the grouping)